ui/package.json

```diff
@@ -10,20 +10,36 @@
     "react-router": "^2.4.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.9.0",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "webpack": "^1.13.1"
+        "@babel/core": "^7.6.4",
```

Comment:
Not that it matters a lot, but the tab size is inconsistent.

Reply:
thanks for catching that - I'll add some formatting help in a follow up PR :)